src/backend/distributed/replication/multi_logical_replication.c

```diff
@@ -1671,94 +1795,36 @@ WaitForRelationSubscriptionsBecomeReady(MultiConnection *targetConnection,
-			{
-				ereport(LOG, (errmsg("Subscription size has been staying same for the "
-									 "last %d msec",
-									 logicalReplicationProgressReportTimeout)));
+			ereport(LOG, (errmsg("Not all subscriptions for the shard move are "
+								 "READY yet")));
 
-				previousReportTime = GetCurrentTimestamp();
-			}
-
-			/* Error out if the size does not change within the given time threshold */
-			if (TimestampDifferenceExceeds(previousSizeChangeTime,
-										   GetCurrentTimestamp(),
-										   LogicalReplicationTimeout))
-			{
-				ereport(ERROR, (errmsg("The logical replication waiting timeout "
-									   "%d msec exceeded",
-									   LogicalReplicationTimeout),
-								errdetail("The subscribed relations haven't become "
-										  "ready on the target node %s:%d",
-										  targetConnection->hostname,
-										  targetConnection->port),
-								errhint(
-									"There might have occurred problems on the target "
-									"node. If not, consider using higher values for "
-									"citus.logical_replication_timeout")));
-			}
+			previousReportTime = GetCurrentTimestamp();
 		}
-		else
-		{
-			/* first, record that there is some change in the size */
-			previousSizeChangeTime = GetCurrentTimestamp();
-
-			/*
-			 * Subscription size may decrease or increase.
-			 *
-			 * Subscription size may decrease in case of VACUUM operation, which
-			 * may get fired with autovacuum, on it.
-			 *
-			 * Increase of the relation's size belonging to subscriber means a successful
-			 * copy from publisher to subscriber.
-			 */
-			bool sizeIncreased = currentTotalRelationSize >
-								 previousTotalRelationSizeForSubscription;
 
-			if (TimestampDifferenceExceeds(previousReportTime,
-										   GetCurrentTimestamp(),
-										   logicalReplicationProgressReportTimeout))
-			{
-				ereport(LOG, ((errmsg("The total size of the relations belonging to "
-									  "subscriptions %s from %ld to %ld at %s "
-									  "on the target node %s:%d",
-									  sizeIncreased ? "increased" : "decreased",
-									  previousReportedTotalSize,
-									  currentTotalRelationSize,
-									  timestamptz_to_str(previousSizeChangeTime),
+		/* Error out if the size does not change within the given time threshold */
+		if (TimestampDifferenceExceeds(previousSizeChangeTime,
+									   GetCurrentTimestamp(),
+									   LogicalReplicationTimeout))
+		{
+			ereport(ERROR, (errmsg("The logical replication waiting timeout "
+								   "%d msec exceeded",
+								   LogicalReplicationTimeout),
+							errdetail("The subscribed relations haven't become "
+									  "ready on the target node %s:%d",
 									  targetConnection->hostname,
-									  targetConnection->port))));
-
-				previousReportedTotalSize = currentTotalRelationSize;
-				previousReportTime = GetCurrentTimestamp();
-			}
+									  targetConnection->port),
+							errhint(
+								"There likely occured problems on the target "
```

Comment:
occured -> occurred

though the sentence is still weird, maybe something like:

Logical replication has failed to initialize on the target node.